libcxx/test/std/ranges/range.adaptors/range.stride.view/concept.verify.cpp

```diff
@@ -0,0 +1,44 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// expected-no-diagnostics
```

Comment:
This shouldn't be a `.verify.cpp`, it should be a `.compile.pass.cpp` instead since you're not expecting any diagnostics. And then this comment should go!